x-pack/metricbeat/tests/system/test_activemq.py

```diff
@@ -0,0 +1,106 @@
+import random
+import string
+import sys
+
+import stomp
+import unittest
+
+from xpack_metricbeat import XPackTest
+import metricbeat
+
+
+class ActiveMqTest(XPackTest):
+    COMPOSE_SERVICES = ['activemq']
+
+    def get_activemq_module_config(self, metricset):
+        return {
+            'name': 'activemq',
+            'metricsets': [metricset],
+            'period': '5s',
+            'hosts': self.get_hosts(),
+            'path': '/api/jolokia/?ignoreErrors=true&canonicalNaming=false',
+            'username': 'admin',
+            'password': 'admin'
+        }
+
+    def get_stomp_host_port(self):
```

Comment:
I wonder why did you need this instead of `compose_host`? I guess this is similar to https://github.com/elastic/beats/pull/14609

If that's the case, it probably makes sense to push it to compose.py

 cc @jsoriano

Reply:
In this case I need to expose two kind of ports. First - REST endpoint with Jolokia and second one - STOMP to communicate with server. Method `compose_host` returns first exposed port of the container. Unfortunately STOMP one is the second.

Reply:
@mtojek `compose_host` can be parameterized with the service and the port you want to get for scenarios with multiple services or multiple ports. In this case for multiple ports you can use `self.compose_host(port='tcp/61613')`. This is used in `test_haproxy.py` for example.

Reply:
Thanks for the suggestion. Fixed!